src/lib.rs

```diff
@@ -209,9 +208,15 @@ mod tests {
         let packed_v = PackedVec::new(v.clone());
         assert_eq!(packed_v.len(), v_len);
         assert_eq!(packed_v.item_width, 33);
-        assert_eq!(packed_v.bits, vec![3221225472, 1073741824,
-                                      4035225266123964416,
-                                      1441151880758558720]);
+        assert_eq!(
```

Comment:
Not sure this is better than the original formatting, but I suppose it's easier if we just accept the style imposed by rustmt. I guess it makes this assert easier to edit.

Reply:
Agreed on both points. But at least it's a standard...